keyboards/AT101_Blackheart/keymaps/default/keymap.c

```diff
@@ -0,0 +1,65 @@
+#include "AT101_Blackheart.h"
```

Comment:
Please change this to `#include QMK_KEYBOARD_H`

Reply:
Fixed.